File: app/src/main/java/bj/discogsbrowser/model/order/Seller.java

package bj.discogsbrowser.model.order;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Seller {

    @SerializedName("resource_url")
    @Expose
    private String resourceUrl;
    @SerializedName("username")
    @Expose
    private String username;
    @SerializedName("id")
    @Expose
    private Integer id;

    public String getResourceUrl() {
        return resourceUrl;
    }

    public void setResourceUrl(String resourceUrl) {
        this.resourceUrl = resourceUrl;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

}

File: app/src/main/java/bj/discogsbrowser/model/common/Seller.kt
package bj.discogsbrowser.model.common

import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class Seller(var username: String = "",
                  @SerializedName("resource_url") var resourceUrl: String = "",
                  var id: String = "",
                  var shipping: String = "",
                  var payment: String = "")